[WIP] some comments (#5)
taylorreiter/2021-paper-ibd: content/03.results.md

We assumed these genomes represent the strongest candidates for discriminating IBD subtype and focused on them for the remainder of our analyses.

![

> **Phylogenetic tree of 360 bacterial species that were predictive of IBD subtype in all models.** Tree was built from the GTDB rs202 tree with all tips except those represented by the 360 genomes removed. Tree tips are labelled by genomes that anchored at least 1% of the normalized variable importance. The inner ring annotates the rank of the genomes, with the genome holding the most normalized variable importance across models ranked as 1. The outer ring is the species name within the GTDB database.  CTB: why this figure; what is the point?

Reviewer: shrug. I feel like this is the eye candy biologists want. any suggestions for what else would go here, vs removal?

I do value having an overview of all of the genomic diversity picked up on by the models, and this was the only summary I could think of that could fit all 360 genomes.

Author: hmm, brainstorming - what about a taxonomic summary instead? something tree something something?

Reviewer: option 1: but colored by percent of variable important (maybe...that might hard. but I could try!). Pros -- shows all species. Would need to figure out how to mark species that were included in the final 54

![image](https://user-images.githubusercontent.com/23057646/165990078-3f1766f0-3244-4b53-a7b6-8b056d830367.png)

option 2: summarize to a taxonomic level. Use a scatter plot to viz the % of variable importance held by those organisms. Use the size of the dot to communicate the number of species per genus. color the dot by the fraction of the final 54 that are in that taxonomic level
![IMG_9470](https://user-images.githubusercontent.com/23057646/165990745-23a59aea-2a0b-444c-9c46-d0d9590ae59f.jpeg)

Author: I kinda like #2, I'm sorry to say :).

Reviewer: cool, current or option 1 will go into supplement, and option 2 will be implemented. Thanks!

merging this PR now, but will leave your comment until I replace the figure

To test this hypothesis, we performed genome queries on assembly graphs of each metagenome using the 54 candidate genomes that discriminated IBD subtype (**Figure @fig:overview**).
Assembly graph genome queries recover sequences in a metagenome that match the query, as well as those that are nearby in the assembly graph, making queries akin to but more sensitive than read mapping against reference genomes (**Figure @fig:overview**) [@doi:10.1186/s13059-020-02066-4].
<!-- is more sensitive the right thing to say here? Should it be more general? -->

> The resulting genome query neighborhood represents a species-level umbrella that contains all sequence variation contained in the metagenome for a query. (CTB: should we say "all" here? How do we know? This sentence seems fishy.)

Author: ```suggestion
The resulting genome query neighborhood represents a species-level umbrella that contains sequence variation from the metagenome associated with a query.
```
how 'bout this?

> The resulting genome query neighborhood represents a species-level umbrella that contains all sequence variation contained in the metagenome for a query. (CTB: should we say "all" here? How do we know? This sentence seems fishy.)

Reviewer: yup that's great, thank you!!
